src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/AppointmentsServiceTest.kt

```diff
@@ -139,6 +140,48 @@ internal class AppointmentsServiceTest {
 
     assertThat(updatedAppointment).isEqualTo(actionPlanAppointment)
     verify(communityAPIBookingService).book(actionPlanAppointment, appointmentTime, durationInMinutes)
+    verify(actionPlanAppointmentRepository).save(
+      ArgumentMatchers.argThat { (
```

Comment:
do we really need this explicit matcher? can't we just match `any()`? i find this code really hard to grep

Reply:
It was the only way I could confirm that the deliusAppointmentId was being persisted.